testhelpers/docker_registry.go

```diff
@@ -96,29 +95,38 @@ func ReadOnly(handler http.Handler) http.Handler {
 	})
 }
 
-func Custom(handler http.Handler, permissions map[string]ImagePrivileges) http.Handler {
+func delegator(basicAuthHandler http.Handler, regHandler http.Handler, permissions map[string]ImagePrivileges) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think that I would change `r` and `w` to `req` and `res` or even to `request` and `response` (or maybe you have better names?).
While working on "read/write" permissions, it's a bit confusing that `w` doesn't influence the permissions because it's just the response writer.

Reply:
Cool, I renamed them in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/c59c94c0966e2826f3e1ce6b48983be810952fee)